javascriptv3/example_code/personalize/README.md

```diff
@@ -0,0 +1,81 @@
+#  Amazon Personalize JavaScript SDK v3 code examples
+The code examples in this directory demonstrate how to work with Amazon Personalize using the AWS SDK for JavaScript version 3 (v3).
+
+Amazon Personalize is a fully managed machine learning service that makes it easy for developers deliver personalized experiences to their users.
+
+## Code examples
+In this workspace, you can find the following AWS SDK for JavaScript version 3 (v3) examples for Amazon Personalize:
+
+### Single action examples
```

Comment:
### Single action

(per the template: https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template)